File: android/TodoCpp/app/src/main/java/com/fhflensburg/todocpp/Logger.java
package com.fhflensburg.todocpp;

import android.util.Log;

/**
 * Author: Dmitri Hammernik
 * Date: 04.04.17
 */

public class Logger extends com.fhflensburg.libtodo.djinni.Logger {
    @Override
    public void i(String tag, String message) {
        Log.i(tag, message);
    }

    @Override
    public void d(String tag, String message) {
        Log.d(tag, message);
    }

    @Override
    public void e(String tag, String error) {
        Log.e(tag, error);
    }
}

File: ios/Todos/Logger.swift
//
//  Logger.swift
//  Todos
//
//  Created by Dmitri Hammernik on 05.04.17.
//  Copyright © 2017 Maik Hansen. All rights reserved.
//

import Foundation

class Logger : FHFLLogger {
    func d(_ tag: String, message: String) {
        print("DEBUG", tag, message)
    }
    
    func i(_ tag: String, message: String) {
        print("INFO", tag, message)
    }
    
    func e(_ tag: String, error: String) {
        print("ERROR", tag, error)
    }
}
